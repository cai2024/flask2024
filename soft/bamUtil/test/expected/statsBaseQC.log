Number of records read = 45
