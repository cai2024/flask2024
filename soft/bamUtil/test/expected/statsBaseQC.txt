chrom	chromStart	chromEnd	TotalReads	Dups	QCFail	Mapped	Paired	ProperPaired	ZeroMapQual	MapQual<10	MapQual255	PassMapQual	AverageMapQuality	AverageMapQualCount	Depth	Q20Bases
1	99	100	3	0	0	3	3	0	1	2	0	3	11.000	3	3	0
1	100	101	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	101	102	3	0	0	3	3	0	1	2	0	3	11.000	3	3	0
1	102	103	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
1	103	104	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
1	104	105	3	0	0	3	3	0	1	2	0	3	11.000	3	3	0
1	105	106	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	106	107	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	107	108	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	108	109	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
1	109	110	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
1	110	111	3	0	0	3	3	0	1	2	0	3	11.000	3	0	0
1	111	112	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	112	113	3	0	0	3	3	0	1	2	0	3	11.000	3	3	3
1	10012	10013	42	6	6	24	21	3	8	16	0	24	11.000	24	21	0
1	10013	10014	39	6	6	21	18	3	7	14	0	21	11.000	21	21	15
1	10014	10015	39	6	6	21	18	3	7	14	0	21	11.000	21	21	3
1	10015	10016	39	6	6	21	18	3	7	14	0	21	11.000	21	0	0
1	10016	10017	39	6	6	21	18	3	7	14	0	21	11.000	21	0	0
1	10017	10018	39	6	6	21	18	3	7	14	0	21	11.000	21	21	0
1	10018	10019	39	6	6	21	18	3	7	14	0	21	11.000	21	21	18
1	10019	10020	39	6	6	21	18	3	7	14	0	21	11.000	21	21	18
1	10020	10021	39	6	6	21	18	3	7	14	0	21	11.000	21	21	18
1	10021	10022	39	6	6	21	18	3	7	14	0	21	11.000	21	0	0
1	10022	10023	39	6	6	21	18	3	7	14	0	21	11.000	21	0	0
1	10023	10024	39	6	6	21	18	3	7	14	0	21	11.000	21	0	0
1	10024	10025	39	6	6	21	18	3	7	14	0	21	11.000	21	21	18
1	10025	10026	39	6	6	21	18	3	7	14	0	21	11.000	21	21	18
